test/pipelinerun_test.go

```diff
@@ -337,6 +478,17 @@ func getHelloWorldPipelineRun(suffix int, namespace string) *v1alpha1.PipelineRu
 	)
 }
 
+func getHelloWorldPipelineRunWithTimeout(suffix int, namespace string) *v1alpha1.PipelineRun {
```

Comment:
since this pipelinerun is specifically about timeouts and retries, can we rename the function and resources to reflect that? (instead of calling it 'hello world')